### PR TITLE
feat: UI completed

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- completed UI
- Added react-router

### CODE REVIEW

**Review by VedanthB — COMMENTED** [review]
Good work, 

things i Liked
- clean structured code 
- good variable naming

You can merge the PR

**Review by VedanthB — APPROVED** [review]

File: src/components/Router.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {Routes, Route} from "react-router-dom";
+import Home from "../pages/home";
+import { Instructions } from "../pages/instructionPage";
+
+export const Router = () => {
+    return (
+        <Routes>
+            <Route path="/" element={<Home />}/>
+            <Route path="/instructions" element={<Instructions />}/>
+            <Route path="/instructions" element={<Instructions />}/>
```
Comment (amarphule) [inline src/components/Router.js:10]: why there are two routes for the same path.

**Review by amarphule — CHANGES_REQUESTED** [review]
Good job, well-structured code.

File: src/components/Router.js (hunk 1 above)
Comment (Vishruta-Patil) [reply src/components/Router.js:10]: Thanks for pointing it out, I forgot to remove that line.

**Pull request merged**